### PR TITLE
Enter key press support for typeahead

### LINKED ISSUE #113: search field: "enter" should bring user to location
For the 1.0 version : choose the location whose name matches "at best" the provided one.

For the alpha version : choose the first one, and if there's no result, display "no result"

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #113.

### CODE REVIEW

**Comment by pgiraud** [comment]
Home page still needs to be updated for styling reasons.

File: thinkhazard/templates/search.jinja2 [hunk 1]
```diff
@@ -1,2 +1,4 @@
 <input id="search-field" class="form-control input-lg" type="text"
        placeholder="Enter location… (e.g. Ben Slimane)">
+<input type="hidden" id="search-division-code" value="" />
+<input type="submit" value="Submit" style="display: none">
```
Comment (elemoine) [inline thinkhazard/templates/search.jinja2]: Does it make sense in this case to set the style in the HTML file?

**Comment by pgiraud** [comment]
Now it's ready for review.

**Comment by elemoine** [comment]
Please merge.

**Pull request merged**